docs/docker/README.md

```diff
@@ -0,0 +1,57 @@
+## TODO for Dockerization
+* Add maintainer(s)
+  - Maintainer for CR Dockerfile needs to be added after pull request is merged. Ideally the maintainer should be someone from official team with appropriate Docker Hub permission to tag image as `box/clusterrunner:latest`. I can volunteer if needed but that will require collaborator permission for Docker Hub.
+* Lean Docker image (consider tiny base e.g. busybox, alpine, etc.)
+  - Currently `Dockerfile.src` uses `python:3.4-slim` as the base. I attempted `python:alpine` as the base but it didn't feel like worth the [trouble](https://github.com/docker/docker/issues/27940). My docker version is `1.12.6` but the problem might have been fixed in `1.13.0+`. I will revisit this in the future.
```

Comment:
Does `python:3.4-slim` include more than what is needed? (I assume it does from your comment, but I'm curious as to what.)